internal/repositories/dcn.go

```diff
@@ -0,0 +1,36 @@
+package repositories
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+
+	gmodel "github.com/DIMO-Network/identity-api/graph/model"
+	"github.com/DIMO-Network/identity-api/models"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func DCNToAPI(d *models.DCN) *gmodel.Dcn {
+	return &gmodel.Dcn{
+		Owner: common.BytesToAddress(d.OwnerAddress),
+		Node:  d.Node,
+	}
+}
+
+func (r *Repository) GetDCNByNode(ctx context.Context, node []byte) (*gmodel.Dcn, error) {
+	var n []byte
```

Comment:
Why should we have to do this again? At this point, `node` should be the bytes we want.